proto/cosmos/crypto/keyring/types.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package cosmos.crypto.keyring.v1;
```

Comment:
```
{"path":"proto/cosmos/crypto/keyring/types.proto","start_line":2,"start_column":1,"end_line":2,"end_column":34,"type":"PACKAGE_DIRECTORY_MATCH","message":"Files with package \"cosmos.crypto.keyring.v1\" must be within a directory \"cosmos/crypto/keyring/v1\" relative to root but were in directory \"cosmos/crypto/keyring\"."}
```

Since I added `v1` to package (proto convention), linter [suggest](url) to move `types.proto` from `*/keyring/.` to `*/keyring/v1` That action will bring additional refactoring work and some complexity since `isRecord_Item`  is private interface defined in `types.proto` and it is used in `record.go`. Therefore, I have to move `/keyring/record.go` and `/keyring/record.go` logic to `keyring/v1` package. Does it refactoring necessary or I can leave as it right now?